database/migrations/0034-drop_server_user_table.ts

```diff
@@ -0,0 +1,37 @@
+/* MIGRATION DROP server_users TABLE
+add isAdmin COLUMN to users TABLE */
+
+/* eslint-disable @typescript-eslint/explicit-module-boundary-types */
+/* eslint-disable @typescript-eslint/no-explicit-any */
+
+export async function upgrade(queryFn: (query: string, values?: any[]) => Promise<Array<any>>) {
+  await queryFn('ALTER TABLE `users` ADD COLUMN `is_admin` datetime DEFAULT NULL AFTER `language`;')
+
+  await queryFn(
+    'UPDATE `users` AS `users`, (SELECT  * FROM `server_users`) AS `server_users` SET users.`is_admin` = server_users.`modified` WHERE users.`email` IN (SELECT email from `server_users`);',
```

Comment:
Does this work? The `(Select * ...)` in the select statement seems dagerously unspecific. I would use a join here - tho this might be a valid solution.

I would do something like this:
` UPDATE users INNER JOIN server_users ON email = email SET is_admin = modified`